Makefile

```diff
@@ -1,38 +1,42 @@
 .DEFAULT_GOAL := help
 .PHONY: help build test
 
+PYPI_TOKEN := $(shell echo ${PYPI_TOKEN})
+PYPI_TEST_TOKEN := $(shell echo ${PYPI_TEST_TOKEN})
+
+
 build: ## Build an application
-	@pipenv run python -m build --no-isolation
+	@poetry build
+
+configure_pypi_publishing: ## Configure publishing to PyPI
+	@if [ -z "${PYPI_TOKEN}" ] ; then echo "you need to export PYPI_TOKEN before running this command" ; false ; fi
+	@if [ -z "${PYPI_TOKEN}" ] ; then echo "you need to export PYPI_TEST_TOKEN before running this command" ; false ; fi
+	@poetry config repositories.test-pypi https://test.pypi.org/legacy/
+	@poetry config pypi-token.test-pypi $(PYPI_TEST_TOKEN)
+	@poetry config pypi-token.pypi $(PYPI_TOKEN)
 
 publish-test: ## Upload package to test PyPI
-	@pipenv run twine upload --repository testpypi dist/*
+	@poetry publish -r test-pypi
 
 publish: build ## Upload package to PyPI
-	@pipenv run twine upload dist/*
+	@poetry publish
 	@make clean
 
-install: build ## Install application to Pip environment
-	@pipenv run python -m pip install
-
-install-dev: ## Install application to Pip development environment
```

Comment:
I'm not very familiar with poetry env, but since you removed this block, does it mean that there is no need to install current package in a dev mode?

Reply:
If its so that `openapi3-parser` will be installed in an editable way, that's already done by poetry.

If it is to allow others to install your package with `pip install -e`, that's also supported by poetry: https://github.com/python-poetry/poetry/issues/34#issuecomment-1193365526

Both are out of the box.